docker/release-ci/Dockerfile_template

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:CYCLUS_UBUNTU_MAJOR_VERSION.04
+
+RUN echo "1"
+
+RUN apt-get -y --force-yes update
+
+RUN apt-get install -y --force-yes \
+    cmake \
+    make \
+    libboost-all-dev \
+    libxml2-dev \
+    libxml++2.6-dev \
+    libsqlite3-dev \
+    libhdf5-serial-dev \
+    libbz2-dev \
+    coinor-libcbc-dev \
+    coinor-libcoinutils-dev \
+    coinor-libosi-dev \
+    coinor-libclp-dev \
+    coinor-libcgl-dev \
+    libblas-dev \
+    liblapack-dev \
+    g++ \
+    libgoogle-perftools-dev \
+    git \
+    python \
+    python-tables \
+    python-numpy \
+    python-nose
```

Comment:
Can't use cyclus-deps, because we need to control the ubuntu version.
but this should be documented...